### PR TITLE
add WAF steps to user documentation

### PR DESCRIPTION
**Pull request by BrianEllwood**
Amend user documentation to add steps for implementing WAF to an application, for work carried out in issue [#1929](https://github.com/ministryofjustice/data-platform/issues/1929)

### CODE REVIEW

**Review by julialawrence — CHANGES_REQUESTED** [review]
Please add a link to a workflow that has these values set.

File: source/documentation/apps/rshiny-app.md [hunk 1]
```diff
@@ -693,3 +693,26 @@ A complete example for installing the helm chart in the workflow below. These ch
...
 A full working example is available [here](https://github.com/ministryofjustice/ap-rshiny-notesbook)
+
+#### Enabling Web Application Firewall (WAF) for Data Platform Apps 
+
+There are two flags that need to be set to enable WAF for your application, Ingress.ModSec.enabled and GithubTeam
+
+To do this by adding the following lines in to your app's  `build-push-deploy-dev.yml` and `build-push-deploy-prod.yml` GitHub action workflow files:
+
+To change the resources insert at  the end of the file just before the $custom_variables the following as appropriate, remembering to use the line
+continuation “\” on the existing last line.
```
Comment (julialawrence) [inline source/documentation/apps/rshiny-app.md:704]:
```suggestion
To do this modify your app's  `build-push-deploy-dev.yml` and `build-push-deploy-prod.yml` GitHub action workflow files as follows:
```

Comment (julialawrence) [reply source/documentation/apps/rshiny-app.md:704]: Made a small tweak to language.

**Review by julialawrence — CHANGES_REQUESTED** [review]
Made a suggestion around language.

**Review by bagg3rs — APPROVED** [review]

**Pull request merged**